src/kaleidoscope/key_defs.h

```diff
@@ -27,7 +27,7 @@
 #include "kaleidoscope/key_defs_aliases.h"
 
 #ifdef ARDUINO_VIRTUAL
-#include "VirtualHID/VirtualHID.h"
+#include "VirtualHID.h"
```

Comment:
The directory `VirtualHID` was renamed `MultiReport`. `VirtualHID.h` was moved one level up in the directory hierarchy.